.github/workflows/test-pr.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       - name: 'Build Foundry'
         run: docker exec -u user kevm-ci-foundry-${GITHUB_SHA} /bin/bash -c 'make build-foundry -j2'
       - name: 'Test Foundry'
-        run: docker exec -u user kevm-ci-foundry-${GITHUB_SHA} /bin/bash -c 'make test-foundry -j2 FOUNDRY_PAR=4 KPROVE_OPTS=--retry'
```

Comment:
I'd suggest removing the `retry` feature since it was only added for CI.

Reply:
It has been removed entirely here.

Reply:
Yes, it was. My bad.